Editor/Integrations/Thunderstore/ThunderstoreAPI.cs

```diff
@@ -19,50 +20,66 @@ public class ThunderstoreAPI
     {
         static string PackageListApi => ThunderKitSetting.GetOrCreateSettings<ThunderstoreSettings>().ThunderstoreUrl + "/api/v1/package";
 
-        internal static List<PackageListing> loadedPackages;
-
         [InitializeOnLoadMethod]
         public static void LoadPages()
```

Comment:
The ThunderstoreAPI package listing updates should be reviewed.  
This is kicking off on a regular basis and may be spamming Thunderstore.
Thunderstore is looking at adding some headers to determine cache update times so we can request headers to check if there are updates and then acquire those updates when needed.

Reply:
this is fine for now per Mythic